Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -50,6 +50,8 @@ def family
             :broadwell
           when 0x37fc219f # Skylake
             :skylake
+          when 0x0f817246 # Kaby Lake
+            :kabylake
```

Comment:
should it be `:kaby_lake`?

Reply:
IMO, yes. But the Linux side of this code & the CPU Hardware tests are already using this style so I just went with it.

Reply:
Yuck.